addons/project/security/ir.model.access.csv

```diff
@@ -36,6 +36,7 @@ access_project_milestone_user,project.milestone.user,model_project_milestone,bas
 access_project_milestone_portal,project.milestone.portal,model_project_milestone,base.group_portal,1,0,0,0
 access_project_milestone_project_user,project.milestone.project.user,model_project_milestone,project.group_project_user,1,0,0,0
 access_project_milestone_project_manager,project.milestone.project.manager,model_project_milestone,project.group_project_manager,1,1,1,1
+access_project_collaborator_user,project.collaborator.manager,model_project_collaborator,project.group_project_user,1,0,0,0
```

Comment:
@xavierbol 'group_project_user' access right's User can not view the project. It will show access right error. see [here](https://drive.google.com/file/d/1o-94gC0W12RRy_oWEqlTFprvj9r5hNx0/view?usp=sharing).